pkg/pravega/bookie.go

```diff
@@ -84,11 +84,11 @@ func makeBookieHeadlessService(pravegaCluster *v1alpha1.PravegaCluster) *corev1.
 	}
 }
 
-func makeBookieStatefulSet(pravegaCluster *v1alpha1.PravegaCluster) *v1beta1.StatefulSet {
-	return &v1beta1.StatefulSet{
+func makeBookieStatefulSet(pravegaCluster *v1alpha1.PravegaCluster) *v1beta2.StatefulSet {
+	return &v1beta2.StatefulSet{
```

Comment:
I think that `apps/v1beta2/StatefulSet` is deprecated in favor of  `apps/v1/StatefulSet`.

Reply:
Updated. Thanks for the heads up.